ui/admin/tests/acceptance/aliases/update-test.js

```diff
@@ -71,26 +59,27 @@ module('Acceptance | aliases | update', function (hooks) {
 
   test('users can update an exisiting alias', async function (assert) {
     await visit(urls.aliases);
-    await click(`[href="${urls.alias}"]`);
-    await click(BUTTON_SELECTOR, 'Click edit mode');
-    await fillIn(NAME_FIELD_SELECTOR, NAME_FIELD_TEXT);
-    await click(SAVE_BTN_SELECTOR);
-    assert.strictEqual(instances.alias.name, NAME_FIELD_TEXT);
+    await click(commonSelectors.HREF(urls.alias));
+    await click(commonSelectors.EDIT_BTN);
+    await fillIn(commonSelectors.FIELD_NAME, commonSelectors.FIELD_NAME_VALUE);
+    await click(commonSelectors.SAVE_BTN);
+    
+    assert.strictEqual(instances.alias.name, commonSelectors.FIELD_NAME_VALUE);
   });
 
   test('can cancel changes to an existing alias', async function (assert) {
     const name = instances.alias.name;
     await visit(urls.aliases);
+    
+    await click(commonSelectors.HREF(urls.aliases));
```

Comment:
Is this click needed? It looks like we are using `visit` to the aliases route, then clicking on it?

Reply:
Interesting that an await click doesn't err if this is not needed. Will comment and verify that tests pass.